polyfill/package.json

```
@@ -5,6 +5,10 @@
   "type": "commonjs",
   "main": "dist/index.js",
   "browser": "dist/index.umd.js",
+  "exports": {
+    "import": "./lib/index.mjs",
+    "require": "./dist/index.js"
+  },
```

this is what would be needed if you wanted to actually support every node version:
```suggestion
  "exports": {
     ".": [
       {
        "import": "./lib/index.mjs",
        "default": "./dist/index.js"
       },
       "./dist/index.js"
     ]
  },
```

but note that this is still a breaking change unless these are the only two files included in the package (which i can see they aren't from the "browser" field above)